.gitignore

```diff
@@ -51,3 +51,4 @@ public/uploads
 public/assets/
 *.env
 docker-compose.yml
+config/initializers/application.yml
```

Comment:
Can you add a template of this file in the PR? Another option would be to add the real file, taking the Sentry API key from the environment.

Reply:
I just added a template, it should make it clearer how to hook things up.